x-pack/legacy/plugins/reporting/server/routes/generate_from_jobparams.ts

```diff
@@ -73,11 +72,10 @@ export function registerGenerateFromJobParams(
     },
   });
 
-  // show error about GET method to user
+  // Get route to generation endpoint: show error about GET method to user
   server.route({
     path: `${BASE_GENERATE}/{p*}`,
     method: 'GET',
-    config: getRouteConfig(),
```

Comment:
This was removed because the routing middleware this adds is not applicable to the `GET` request for this.